sdformat_rs/build.rs

```diff
@@ -53,7 +53,7 @@ impl RequiredStatus {
     fn from_str(required: &str) -> RequiredStatus {
         if required == "true" || required == "1" {
             return RequiredStatus::One;
-        } else if required == "*" {
+        } else if required == "*" || required == "+" {
```

Comment:
I can't recall why I didn't do this but there was some reason... I'm not sure what `+` means. Does it mean `AtLeastOne`?

Reply:
Indeed that seems to be the case (i.e. from [here](https://github.com/gazebosim/sdformat/blob/24b61531ad49420aeddb9f59c4c08b01b59e5d29/tools/xmlschema.rb#L53-L55)).
We can either leave this detail out and just keep a `Vec` or use some external non empty container (i.e. the [nonempty crate](https://crates.io/crates/nonempty)).
What would your preference be?